n.cash complaints (#5)
Anastasia-Romanov/lacan-user-guide: nitrotype.md

## cash

- It requires about a trillion arguments in a specific order which is quite easy to forget.  An actual UI (such as slash commands) would greatly improve this command.

> - Using an accuracy below 0 or greater than 100 results in an inaccurate error statement to use "whole, rounded numbers" even with 101, 102, etc

Reviewer: ```suggestion
- Using an accuracy below 0 or greater than 100 results in an inaccurate error statement to use "whole, rounded numbers" even with 101, 102, etc.
```

> - Decimal amount of friends means you have none.

Reviewer: ```suggestion
- A decimal amount of friends is interpreted as you having none.
```

> - apparentally `n.cash 0 0 0 0 False piss` doesn't get an error smh my head

Reviewer: ```suggestion
- Apparently `n.cash 0 0 0 0 False piss` doesn't get an error smh my head.
```

> - An actual UI (such as slash commands) has options to only allow intergers or booleans.

Reviewer: ```suggestion
- An actual UI (such as slash commands) has options to only allow integers or booleans.
```
